docs/source/extending/frontend_extensions.rst

```diff
@@ -219,11 +219,11 @@ rather than copying it, so there's no need to reinstall after changes.
 To use your extension, you'll also need to **enable** it, which tells the
 notebook interface to load it. You can do that with another command::
 
-    jupyter nbextension enable my_extension/main [--sys-prefix]
+    jupyter nbextension enable my_extension/main [--sys-prefix|--section='common']
```

Comment:
It's not mutually exclusive from the `--sys-prefix` option, so the normal way is to put each one in its own `[square brackets]`.